dotnet/src/Agents/AzureAI/AzureAIAgent.cs

```diff
@@ -186,6 +186,19 @@ public async IAsyncEnumerable<AgentResponseItem<ChatMessageContent>> InvokeAsync
             () => new AzureAIAgentThread(this.Client),
             cancellationToken).ConfigureAwait(false);
 
+        var kernel = (options?.Kernel ?? this.Kernel).Clone();
+
+        // Get the context contributions from the AIContextBehaviors.
+#pragma warning disable SKEXP0110 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+        var extensionsContext = await azureAIAgentThread.AIContextBehaviors.OnModelInvokeAsync(messages, cancellationToken).ConfigureAwait(false);
```

Comment:
`extensions` was not clear for me, suggest changing the name

```suggestion
        var behaviorContext = await azureAIAgentThread.AIContextBehaviors.OnModelInvokeAsync(messages, cancellationToken).ConfigureAwait(false);
```

Reply:
Makes sense, changed to BehaviorsContext.